tests/helpers.py

```diff
@@ -760,13 +760,51 @@ def measure_pvc_creation_time(interface, pvc_name):
     start = [
         i for i in logs if re.search(f"provision.*{pvc_name}.*started", i)
     ][0].split(' ')[1]
-    # Extract the end time for the PVC provisioning
+    return datetime.datetime.strptime(start, format)
+
+
+def get_end_creation_time(interface, pvc_name):
+    """
+    Get the ending creation time of a PVC based on provisioner logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_name (str): Name of the PVC for creation time measurement
+
+    Returns:
+        datetime object: Creation time for the PVC
```

Comment:
datetime object: end time for the PVC creation

Reply:
done